modules/security-context-constraints-about.adoc

```diff
@@ -21,6 +21,11 @@ Security context constraints allow an administrator to control:
 * The usage of volume types
 * The configuration of allowable `seccomp` profiles
 
+[IMPORTANT]
+====
+Do not set the `openshift.io/run-level` label on any namespaces in {product-title}. This label is for use by internal {product-title} components to manage the startup of major API groups, such as the Kubernetes API server and OpenShift API server. If the `openshift.io/run-level` label is set, no SCCs are applied to pods in that namespace, causing any workloads running in that namespace to be highly privileged.
```

Comment:
@yaoli-redhat Can you review this warning message?

All other files in this PR were reviewed by the other relevant teams. Thanks!

Preview: https://deploy-preview-35852--osdocs.netlify.app/openshift-enterprise/latest/authentication/managing-security-context-constraints?utm_source=github&utm_campaign=bot_dp#security-context-constraints-about_configuring-internal-oauth

Reply:
LGTM